source/client-side-encryption/client-side-encryption.rst

```diff
@@ -838,8 +844,27 @@ Identifies a key vault collection document by 'keyAltName'.
 
 algorithm
 ^^^^^^^^^
-The string "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic" or
-"AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+One of the strings:
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+- "IndexedEquality"
```

Comment:
I think we should either just name this "indexed" or get rid of queryType?

Since we will add Range support, would uses specify `queryType =QueryTypeRanged` and `algorithm = IndexedEquality` in that case?

Reply:
Range will require a new payload. The Range payload is a set of values.